internal/services/v1/permissions.go

```diff
@@ -393,158 +393,7 @@ func TranslateExpansionTree(node *core.RelationTupleTreeNode) *v1.PermissionRela
 	}
 }
 
-const lrv2CursorFlag = "lrv2"
```

Comment:
keep the cursor flag; we might need it again in the future and we don't want to break existing cursors. we should just return an error if the cursor does not have it

Reply:
I don't see this change reverted